ghost/admin/app/components/gh-context-menu.js

```diff
@@ -74,4 +128,36 @@ export default class GhContextMenu extends Component {
     stopClicks(event) {
         event.stopPropagation();
     }
+
+    @task
+    *confirmWrapperTask(...args) {
```

Comment:
This had to be a method and use instance variables rather than an inline task and context variables because for some reason Ember was not happy about inline tasks.